### ISSUE TITLE
Import Error: No module named xxx

### ISSUE THREAD

**Issue by travBoog** [seanharr11/etlalchemy#1]
Tried on RHEL 7, macOS, and Windows 7 running Anaconda env's 3.4.4 and 3.5 all result in same error.

```
from etlalchemy import ETLAlchemySource, ETLAlchemyTarget
import pymysql
import cx_Oracle

o_db = ETLAlchemySource("db connection info")
m_db = ETLAlchemyTarget("db connection info")
target.addSource(source)
target.migrate()
```
results in

ImportError:  No module named 'ETLAlchemySource'
Import Error: No module named 'ETLAlchemyTarget'

**Comment by travBoog** [comment]
I think there is an issue with the ```__init__.py```.  I deleted the original and created a new one using nano and I am now able to get past the ```__init__.py```.  I now am failing on 

```
    from StringIO import StringIO
ImportError: No module named 'StringIO'
```

**Comment by travBoog** [comment]
I can import it now will commit changes and request pull

**Comment by seanharr11** [comment]
Closing - this has been addressed on [the Python 3.5 branch](https://github.com/seanharr11/etlalchemy/tree/python3.5-support)
